kernel/init.c

```diff
@@ -74,7 +74,6 @@ K_KERNEL_STACK_ARRAY_DEFINE(z_interrupt_stacks, CONFIG_MP_NUM_CPUS,
 
 extern void idle(void *unused1, void *unused2, void *unused3);
 
-
```

Comment:
@mmahadevan108 can you remove the white space diff from init.c? that is going to remove some of the review requests, as well :)

Reply:
Thank you for catching this. This was accidentally added.